src/data/contractTX.js

```diff
@@ -136,4 +137,22 @@ export const TX = {
     errMsg: 'Error Processing Proposal',
     successMsg: 'Proposal Processed!',
   },
+  WITHDRAW_PROPOSAL: {
+    contract: 'Moloch',
+    name: 'withdrawBalance',
+    poll: 'subgraph',
+    onTxHash: ACTIONS.BASIC,
+    display: 'Withdraw Balance',
+    errMsg: 'Error Withdrawing Balance',
+    successMsg: 'Balance Withdrawn!',
+  },
+  RAGE_QUIT_PROPOSAL: {
```

Comment:
this name is confusing - it's not a proposal - so let's just use 'RAGE_QUIT'

Reply:
Oh yeah .. my bad .. Will change it :)